### DISCUSSION TITLE
Why is the keyboard not displayed when controlling the phone remotely?

### DISCUSSION THREAD

**Discussion by 1459586433** [rustdesk/rustdesk#8309] [General]
### Bug Description

When the computer/mobile phone remotely controls the mobile phone, click the dialog box and the keyboard will not be displayed. The keyboard can pop up but the content will not be displayed.

### How to Reproduce

When the computer/mobile phone remotely controls the mobile phone, click the dialog box and the keyboard will not be displayed. The keyboard can pop up but the content will not be displayed.

### Expected Behavior

When the computer/mobile phone remotely controls the mobile phone, click the dialog box and the keyboard will not be displayed. The keyboard can pop up but the content will not be displayed.

### Operating system(s) on local side and remote side

windows11

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

When the computer/mobile phone remotely controls the mobile phone, click the dialog box and the keyboard will not be displayed. The keyboard can pop up but the content will not be displayed.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/releases/tag/nightly
